policyengine_us/parameters/gov/states/ca/tax/income/credits/foster_youth/base.yaml

```
@@ -0,0 +1,31 @@
+description: California provides the following foster youth tax credit amount, based on age.  
+brackets:
+  - threshold:  
+      2022-01-01: 0  
+    amount:  
+      2022-01-01: 0  
+  - threshold:  
+      2022-01-01: 18  
+    amount:  
+      2022-01-01: 1_083
+  - threshold:  
+      2022-01-01: 25  
```

available to 18-25 right?
```suggestion
      2022-01-01: 26
```